scripts/pytest-tests.sh

```diff
@@ -30,24 +34,18 @@ cd "$SRCDIR/test/python" && source ./venv/bin/activate
 
 run_tests << 'END'
 
-test_bdd_pool
-test_bdd_replica
-test_bdd_nexus
-test_nexus_publish
-test_bdd_rebuild
-test_bdd_csi
-test_csi
-
-test_ana_client
-test_cli_controller
-test_replica_uuid
-# test_rpc
-
-test_bdd_nexus_multipath
-test_nexus_multipath
-
-test_multi_nexus
-test_nexus
-test_null_nexus
+tests/replica
+tests/publish
+tests/rebuild
+
+# tests/csi
+
+tests/ana_client
+tests/cli_controller
+tests/replica_uuid
```

Comment:
Should the replica_uuid tests also come under `tests/replica`?

Reply:
This is indeed possible from a technical perspective. I have kept them as "logically" distinct for now as they were authored by different people. As mentioned below, there are other minor changes to this test that perhaps need addressing. If so I would prefer that "moving" the test be done at that time instead of here and now